manim/_config/logger_utils.py

```diff
@@ -88,7 +94,7 @@ def make_logger(parser, verbosity):
     return logger, console
 
 
-def parse_theme(parser):
+def parse_theme(parser: configparser.ConfigParser) -> Theme:
```

Comment:
If you see 
https://github.com/ManimCommunity/manim/blob/f1e0c329e74cb62969273e288a160a407e9f0216/manim/_config/logger_utils.py#L24
 have import `from rich.theme import Theme` so this would be correct rather then the one you suggested.